pkgs/applications/graphics/paraview/default.nix

```diff
@@ -1,6 +1,6 @@
 {
-stdenv, fetchFromGitHub, cmake
-,qtbase, qttools, python, libGLU_combined
+stdenv, fetchFromGitHub, cmake, makeWrapper
+,qtbase, qttools, python, pythonPackages, libGLU_combined
```

Comment:
drop `python`

Reply:
Actually, you should probably keep `python` and drop `pythonPackages`. You can use `python.pkgs` instead.